refresh-data.sh

```diff
@@ -42,7 +42,7 @@ if [[ -z "$refresh_dump_name" ]]; then
 
 Or you can define the location of a dump and this script will download it for you:
 
-	export CFGOV_PROD_DB_LOCATION=http://some-bucket.s3.amazonaws.com/wherever/production_django.sql.gz
+	export CFGOV_PROD_DB_LOCATION=https://some-bucket.com/wherever/production_django.sql.gz
```

Comment:
Now that we're removing references to our explicit use of S3, how would you feel about removing the word "bucket" here? Since in theory the production dumps could live anywhere.

Reply:
Updated to `example.com`